README.md

```diff
@@ -132,13 +132,29 @@ Hubspot::Deal.create!(nil, [company.vid], [contact.vid], pipeline: 'default', de
 
 ### Testing
 
-All tests can be run with `rake spec`. Isolate fast-running tests with `rake spec:quick`.
+This project uses [VCR] to test interactions with the HubSpot API.
+VCR records HTTP requests and replays them during future tests.
 
-GET requests are pretty easy to test with VCR, but for POST/PUT requests, you probably want to update verify the state
-of a live HubSpot instance. To do this, please add "live" tests to `spec/live/`, using the rspec label `live: true` in
-order to disable VCR.
+All tests can be run with `rake spec`.
 
-"Live" tests can be isolated with `rake spec:live`.
+Tests under the `live` directory do not use VCR and issue real network calls.
+These tests are being migrated over to use VCR.
+
+By default, the VCR recording mode is set to `:none`, which allows recorded
+requests to be re-played but raises for any new request. This prevents the test
+suite from issuing unexpected HTTP requests.
+
+When adding a new test or updating a VCR recording, run the test with the
+appropriate `VCR_RECORDING` prefix:
+
+```ruby
+VCR_RECORD=new_episodes bundle exec rspec spec
```

Comment:
I like this idea although I think you could just leave the environment variable as `VCR_RECORD` since it's already in active tense. I think it's an unlikely use case that anyone would be looking to set a specific recording mode from the environment variable. It's a more likely use case that a developer has just added some new tests with new API calls and wants to generate the cassettes before completing the work and committing the tests.

Reply:
I dig it 👍 I think it's helpful to view the vcr recording mode as either "on" or "off".

Reply:
@tjmw @cbisnett I've updated to include your feedback 45f69849705fd9eda2e00d97708b044d9de5f4ad